### PR TITLE
Shrevin: PLAN.md updates

### PR DESCRIPTION
**Pull request by shrevin**
I added a basic outline to the PLAN.md file. Can someone review it so we can push it to master?

### CODE REVIEW

**Review by AnushreeTib — CHANGES_REQUESTED** [review]
I think there's a typo in Sprint 1. Did you mean "Build basic model using Google Maps API..."

**Comment by shrevin** [comment]
Ok, fixed the typo! Can you please merge it to the master branch?

File: Docs/PLAN.md [hunk 1]
```diff
@@ -1 +1,34 @@
-Plan file. Can also be a PDF or something else. Just make sure to name PLAN with all caps
+Shark Byte
+
+Sprint 0
+MVP
+Plan.md
+
+Sprint 1
+
+Get familiar with Python and tech stack
```
Comment (stuti-agrawal) [inline Docs/PLAN.md:9]: Let's mention which tech stack we are using.

File: Docs/PLAN.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+Work on frontend
+
+Sprint 3
+
+Finalize MVP code
+
+Sprint 4
+
+Split up work to work on individual features
```
Comment (stuti-agrawal) [inline Docs/PLAN.md:25]: Shouldn't we be splitting up work at the beginning?

**Pull request merged**